onnxruntime/core/providers/dml/DmlExecutionProvider/src/precomp.h

```diff
@@ -53,6 +54,10 @@
 #include "External/DirectMLHelpers/SchemaHelpers.h"
 #include "External/DirectMLHelpers/GeneratedSchemaHelpers.h"
 #include "External/DirectMLHelpers/DirectMLX.h"
+#include "External/DirectMLHelpers/DmlSerializedGraphDesc.h"
+#include "flatbuffers/flatbuffers.h"
```

Comment:
One of these things is unlike all the others. It's more akin to the general headers above this block of "// DirectML helper libraries".